plugins/dell-dock/dell-dock.quirk

```diff
@@ -63,10 +63,10 @@ Icon = dock-usb
 FirmwareSizeMin = 0x1FFC0
 FirmwareSizeMax = 0x20000
 Flags = require-ac,dual-image,self-recovery,usable-during-update
-Children = FuDellDockStatus|USB\VID_413C&PID_B06E&hub&status,FuDellDockMst|MST-panamera-vmm5331-259
 DellDockUnlockTarget = 1
 DellDockBoardMin = 6
 DellDockVersionLowest = 01.00.00.00
+DellDockUSB4VersionLowest = 00.00.00.01
```

Comment:
In development stage

Reply:
Shouldn't this version be later than 01.00.00.00 though?

Reply:
they start from 0.x  :)

Reply:
Removed `DellDockUSB4VersionLowest` as needn't.